src/style.css

```diff
@@ -0,0 +1,37 @@
+html {
```

Comment:
The 'html' selector should be replaced with 'body' as the margin property does not apply to the 'html' element.